test/controllers/pledge_controller_test.rb

```diff
@@ -0,0 +1,89 @@
+require 'test_helper'
+
+class PledgeControllerTest < ActionController::TestCase
+  
+  before do
+    @user = create :user
+    sign_in @user
+    @patient = create :patient
+    @pregnancy = create :pregnancy, patient: @patient
+  end
+  
+  describe 'create method' do
+    before do
+      @pledge = create :pledge,
```

Comment:
Can we use `attributes_for :pledge` instead of `create :pledge` here? That way, we'll have the attributes we want to post without creating duplicate objects in the db.